packages/upswyng-web/src/components/HotlineCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from "@material-ui/core";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+
+const initiatePhoneCall = (phoneNumber: string): void => {
+  window.open(`tel:${phoneNumber}`);
+};
+
+interface Props {
+  name: string;
+  description: string;
+  phone: string;
+}
+
+const HotlineCard = ({ name, description, phone }: Props) => {
+  const [hideText, setHideText] = React.useState(true);
+  return (
+    <ListItem button>
+      <ListItemText
+        onClick={() => initiatePhoneCall(phone)}
```

Comment:
```suggestion
        component={Link}
        to={`tel${phone}`}
```
We can import `Link` from `react-router-dom` and use `ListItem`'s `component` prop to avoid needing this extra fuction.

Reply:
While I totally love this suggestion, It does make it impossible to click the "more" and "less" buttons if we put the functionality on the ListItem rather than the ListItemText

Reply:
ah, no worries then good as is 👍